diff --git a/cnijnpr/src/cnijnpr.c b/cnijnpr/src/cnijnpr.c
index a067bae..b2db55a 100755
--- a/cnijnpr/src/cnijnpr.c
+++ b/cnijnpr/src/cnijnpr.c
@@ -584,7 +584,7 @@ err01:
 /************************************************/
 /*             get_printer_devid               */
 /************************************************/
-static int get_printer_devid(fdusb)
+static int get_printer_devid(int fdusb)
 {
 	char	devid[1024];
 	char	*ivec = NULL;
